### DISCUSSION TITLE
Add `apple pkl` and `nix` icon

### DISCUSSION THREAD

**Discussion by Eveeifyeve** [withastro/starlight#1599] [Feature Requests]
### What version of `starlight` are you using?

0.21.1

### What is your idea?

To add a new language json/yaml killer that was hyped in the community called pkl it is by apple and adds the os/package-manager/lang called nix and it's used by the wider linux community it offers a lot of customisation and flexibility.

### Why is this feature necessary?

It will be useful for in the new tab icons in `<TabItem` component. not only that people could document how the schema is setup for `apple pkl` and then nix you can have documentation for NixOS config and flakes.

### Do you have examples of this feature in other projects?

Have a repo here ready for any PR for this feature: https://github.com/Eveeifyeve/starlight/tree/hd-nix-pkl-icons

- Nix: https://nixos.wiki/
- Applepkl: https://pkl-lang.org/index.html

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by Eveeifyeve** [comment]
#1628 Added the nix icons just apple pkl icon

**Comment by delucis** [comment]
The Pkl icon was just added in Starlight v0.21.5 — thanks again for the request!

**Reply by Eveeifyeve** [reply]
Awesome.
